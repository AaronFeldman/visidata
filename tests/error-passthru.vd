sheet	col	row	keystrokes	input	comment
			o	sample_data/sample.tsv	open file
sample	Item	2	,		
sample	Item		gzd		
sample	Item		=	Item.foo	
sample	Item.foo		F		
