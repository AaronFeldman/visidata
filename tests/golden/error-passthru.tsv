Quantity.foo	count	percent	histogram
'str' object has no attribute 'foo'	49	100.00	************************
